include/odp/api/spec/packet_io_types.h

```diff
@@ -377,6 +377,29 @@ typedef union odp_pktin_config_opt_t {
 		  * on packet input */
 		uint64_t ts_ptp        : 1;
 
+		/** Strip VLAN tags of received packets so that the ODP
+		 *  application sees untagged packets. VLAN ID is saved
+		 *  in packet metadata and can be read using
+		 *  odp_packet_vlan_info().
+		 *
+		 *  This removes only the outermost VLAN tag and only if
+		 *  its ethertype is 0x8100.
+		 *
+		 *  VLAN stripping is done as a last step of packet input
```

Comment:
Discussed internally with Kiran and Jerin. 

Basically this statement is fine with us as for Inline IPsec, though the VLAN Tag will be available for parsing and classification, it will be retained in original packet where it got decrypted.

Reply:
While for non Inline IPsec packets, it will be stripped post parsing.